appyx-navigation/common/src/commonTest/kotlin/com/bumble/appyx/navigation/node/NodeTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.bumble.appyx.navigation.node
+
+import com.bumble.appyx.helpers.RootNode
+import com.bumble.appyx.navigation.modality.NodeContext
+import com.bumble.appyx.utils.multiplatform.SavedStateMap
+import kotlin.test.Test
+import kotlin.test.assertNotEquals
+
+class NodeTest {
+    @Test
+    fun WHEN_node_is_create_THEN_plugins_are_setup_as_expected() {
```

Comment:
let's add a test that verifies that `onSavedInstanceState` is called on AppyxComponent when Node is saving its state?